services/QuillLMS/engines/evidence/spec/dummy/db/schema.rb

```diff
@@ -248,15 +248,6 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
-  create_table "evidence_rule_hints", force: :cascade do |t|
```

Comment:
Just want to confirm: this is an artifact of `evidence_rule_hints` moving to `evidence_hints`, hosted in the umbrella app, not the engine?

Reply:
I'm removing this file -- it should not be in this PR and was accidentally added in.